staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -208,6 +208,8 @@ type SecureServingInfo struct {
 	// BindNetwork is the type of network to bind to - defaults to "tcp", accepts "tcp",
 	// "tcp4", and "tcp6".
 	BindNetwork string
```

Comment:
is BindNetwork still used?

Reply:
no

Reply:
What about the address and port?

Reply:
I will check again

Reply:
BindAddress is "ip:port", it is used.
```
func (s *SecureServingInfo) NewLoopbackClientConfig(token string, loopbackCert []byte) (*restclient.Config, error) {
	if s == nil || (s.Cert == nil && len(s.SNICerts) == 0) {
		return nil, nil
	}

	host, port, err := LoopbackHostPort(s.BindAddress)
```

Reply:
but it's implicit in the listener I would expect.

Reply:
Understand. Resolve it from listener.

Reply:
It is also used by http.Server.Addr both insecure and secure server. So at least four different places, Really worth remove this `BindAddress`?

Reply:
I resolved it.